installer/scripts/tue-install-impl.bash

```diff
@@ -467,6 +468,20 @@ function tue-install-snap
 
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
+function tue-install-dpkg
+{
+    if [ -z "$1" ]
+    then
+        tue-install-error "Invalid tue-install-dpkg call: needs package as argument."
+    fi
+    tue-install-debug "Installing dpkg $1"
+    sudo dpkg --install $1
```

Comment:
Maybe check if file exist?

Reply:
The error thrown by dpkg is:
```bash
dpkg: error processing archive testszz (--install):
 cannot access archive: No such file or directory
```
Would that suffice?

Reply:
Yep I think so.